docassemble/ALToolbox/al_income.py

```diff
@@ -271,7 +325,12 @@ def init(self, *pargs, **kwargs):
         super().init(*pargs, **kwargs)
         self.object_type = ALJob
 
-    def total(self, times_per_year=1, source=None):
+    def total(
+        self,
+        times_per_year: float = 1,
+        source: Union[List[str], str] = None,
+        owner: str = None,
```

Comment:
Though `owner` is up here, it's not passed into the call of `self.gross_total()` in the function.

Reply:
`mypy` complained that we inherited the `total()` method from ALIncomeList but didn't use the same method signature. The `owner` attribute is only used for error-free typing. I agree it's a bit weird for this method.